core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -150,6 +146,11 @@ namespace kagome::consensus::grandpa {
         }
       }
     });
+    return true;
+  }
+
+  bool GrandpaImpl::start() {
+    // Obtain last completed round
```

Comment:
That's a pretty confusing change.

Reply:
yep. it's discussed with @xDimon. We decided to move some logic into preparation instead of fixing call order of apps, call grandpa->start before babe->start.